internal/app/machined/pkg/system/services/kubelet.go

```diff
@@ -105,6 +105,11 @@ func (k *Kubelet) DependsOn(runtime.Runtime) []string {
 	return []string{"cri"}
 }
 
+// Volumes implements the Service interface.
+func (k *Kubelet) Volumes() []string {
+	return nil
```

Comment:
same with kubelet, no ephemeral

Reply:
same here, it needs not EPHEMERAL, but rather some other volumes which I don't have yet